langroid/agent/task.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         default_human_response: Optional[str] = None,
         only_user_quits_root: bool = True,
         erase_substeps: bool = False,
+        rich_log: bool = True,
```

Comment:
We shouldn't add another param in the `Task` constructor. Better to add a `rich_log(enable=True)` method